catalog/src/main/res/values/strings.xml

```diff
@@ -21,7 +21,7 @@
   OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
   SOFTWARE.
   -->
-<resources xmlns:tools="http://schemas.android.com/tools" tools:locale="en">
+<resources xmlns:tools="http://schemas.android.com/tools" tools:locale="en-GB">
```

Comment:
I don't get it. We are adding a new file for British English specifically, but we are still changing the default to the same locale.

Reply:
I've removed it, I thought this impacted the suggested language but no.

Reply:
Ok, but I don't think this is how it works. This is only a tools attribute, meaning it probably has no impact on the app itself.
When trying to load the specific en-gb locale, it will not find any, and will fallback to the default one.
That means the en-gb locale does not add anything but an extra entry in the locale list. Or am I missing something?

Reply:
I didn't remove the tools label but the new (extra) values-en-rGB, I wasn't explicit sorry.
You're correct this only help for lint in this file and doesn't help at all for resolving which locale is available